src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -85,68 +94,106 @@ def compute_rank_from_scores(
     # The expected rank of a random scoring
     expected_rank = 0.5 * (number_of_options + 1)
```

Comment:
maybe we should make this more explicit that it's the `expected_realistic_rank`. I'm a bit worried about conflating the "mean" in mean rank with the "average" between the best/worst. Plus, in the tutorial, this is called the "realistic" ranking

Reply:
I think we should stick to the same word familiy for the variants, so either
```
best / worst / avg

optimistic / pessimistic / realistic
```

Reply:
I like `optimistic / pessimistic / realistic` better

Reply:
I switched them all. do you think this is going to affect a lot of people who are relying on the old keys?

Reply:
I am not sure. We had avg being the default rank type, right? And I guess only few people explicitly said they want to have early stopping based on best / worst. So my guess is that it will not affect many.

Reply:
gucci